apps/api/src/app/workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase.ts

```diff
@@ -184,28 +185,13 @@ export class UpsertWorkflowUseCase {
       description: workflowDto.description || '',
       tags: workflowDto.tags || [],
       critical: false,
+      triggerIdentifier: `${slugify(workflowDto.name, {
+        lower: true,
+        strict: true,
+      })}`,
     };
   }
 
-  private async getWorkflowIfUpdateAndExist(upsertCommand: UpsertWorkflowCommand) {
-    if (upsertCommand.workflowDatabaseIdForUpdate) {
-      return await this.notificationTemplateRepository.findByIdQuery({
-        id: upsertCommand.workflowDatabaseIdForUpdate,
-        environmentId: upsertCommand.user.environmentId,
-      });
-    }
-  }
-
-  private async workflowExistByExternalId(upsertCommand: UpsertWorkflowCommand) {
-    const { environmentId } = upsertCommand.user;
-    const workflowByDbId = await this.notificationTemplateRepository.findByTriggerIdentifier(
-      environmentId,
-      upsertCommand.workflowDto.name
-    );
-
-    return !!workflowByDbId;
-  }
```

Comment:
we don't need to validate because this logic is handled in the Create/Update workflow application-generic logic. so this call is duplicated.
(small note we enforce it on the database layer as well)